vars/gh.groovy

```diff
@@ -87,8 +89,8 @@ def runCommand(command, flagsCommand) {
   return output
 }
 
-def downloadInstaller(where) {
-  def url = 'https://github.com/cli/cli/releases/download/v1.9.2/gh_1.9.2_linux_amd64.tar.gz'
+def downloadInstaller(where, version) {
+  def url = "https://github.com/cli/cli/releases/download/v${version}/gh_${version}_linux_amd64.tar.gz"
```

Comment:
NIT: on macOS will fail to execute the binary.

Reply:
I'll do a follow up